utilities/rpp-unittests/OCL/BatchPD_ocl.cpp

```diff
@@ -0,0 +1,729 @@
+        break;
+    case 6:
+        test_case_name = "gamma_correction";
+        rppi_gamma_correction_u8_pkd3_batchPD_gpu(d_input, srcSize, maxSize, d_output, gamma, noOfImages, handle);
+        break;
+    case 7:
+        test_case_name = "fog";
+        rppi_fog_u8_pkd3_batchPD_gpu(d_input, srcSize, maxSize, d_output, fogValue, noOfImages, handle);
+        break;
+    case 8:
+        test_case_name = "snow";
+        rppi_snow_u8_pkd3_batchPD_gpu(d_input, srcSize, maxSize, d_output, snowPercentage, noOfImages, handle);
+        break;
+    case 9:
+        test_case_name = "lens_correction";
+        rppi_lens_correction_u8_pkd3_batchPD_gpu(d_input, srcSize, maxSize, d_output, strength, zoom, noOfImages, handle);
+        break;
+    case 10:
+        test_case_name = "noise";
+
+        rppi_noise_u8_pkd3_batchPD_gpu(d_input, srcSize, maxSize, d_output, noiseProbability, noOfImages, handle);
+
+        break;
+    case 11:
+        test_case_name = "pixelate";
+        rppi_pixelate_u8_pkd3_batchPD_gpu(d_input, srcSize, maxSize, d_output, noOfImages, handle);
+        break;
+    case 12:
+        test_case_name = "exposure";
+        rppi_exposure_u8_pkd3_batchPD_gpu(d_input, srcSize, maxSize, d_output, exposureFactor, noOfImages, handle);
+        break;
+    case 13:
+        test_case_name = "fisheye";
+        rppi_fisheye_u8_pkd3_batchPD_gpu(d_input, srcSize, maxSize, d_output, noOfImages, handle);
+        break;
+    case 14:
+        test_case_name = "vignette";
+        rppi_vignette_u8_pkd3_batchPD_gpu(d_input, srcSize, maxSize, d_output, stdDev, noOfImages, handle);
+        break;
+    case 15:
+        test_case_name = "flip";
+        rppi_flip_u8_pkd3_batchPD_gpu(d_input, srcSize, maxSize, d_output, flipAxis, noOfImages, handle);
+        break;
+    case 16:
+        test_case_name = "rain";
+        rppi_rain_u8_pkd3_batchPD_gpu(d_input, srcSize, maxSize, d_output, rainPercentage, rainWidth, rainHeight, transparency, noOfImages, handle);
+        break;
+    case 17:
+        test_case_name = "rotate";
+        rppi_rotate_u8_pkd3_batchPD_gpu(d_input, srcSize, maxSize, d_output, dstSize, maxSize, angle, noOfImages, handle);
+        break;
+    case 18:
+        test_case_name = "warp-affine";
+        rppi_warp_affine_u8_pkd3_batchPD_gpu(d_input, srcSize, maxSize, d_output, dstSize, maxSize, affine_array, noOfImages, handle);
+        break;
+
+    case 19:
+        test_case_name = "resize";
+        rppi_resize_u8_pkd3_batchPD_gpu(d_input, srcSize, maxSize, d_output, dstSize, maxDstSize, noOfImages, handle);
+        break;
+
+    case 20:
+        test_case_name = "resize-crop";
+        rppi_resize_crop_u8_pkd3_batchPD_gpu(d_input, srcSize, maxSize, d_output, dstSize, maxDstSize, x1, x2, y1, y2, noOfImages, handle);
+        break;
+    case 21:
+        test_case_name = "Hue modification";
```

Comment:
Codacy found an issue: [Variable 'test_case_name' is assigned a value that is never used.](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/rpp/pullRequest?prid=5976402)